[2025-10-11T06:11:38.193+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-10-11T06:11:38.243+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: pipeline_flow.load_ratings_to_gold scheduled__1999-03-28T00:00:00+00:00 [queued]>
[2025-10-11T06:11:38.256+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: pipeline_flow.load_ratings_to_gold scheduled__1999-03-28T00:00:00+00:00 [queued]>
[2025-10-11T06:11:38.258+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2025-10-11T06:11:38.280+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): load_ratings_to_gold> on 1999-03-28 00:00:00+00:00
[2025-10-11T06:11:38.292+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:62 DeprecationWarning: This process (pid=10941) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-10-11T06:11:38.294+0000] {standard_task_runner.py:64} INFO - Started process 10945 to run task
[2025-10-11T06:11:38.293+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'pipeline_flow', 'load_ratings_to_gold', 'scheduled__1999-03-28T00:00:00+00:00', '--job-id', '963', '--raw', '--subdir', 'DAGS_FOLDER/pipeline_dag.py', '--cfg-path', '/tmp/tmptk4_gj22']
[2025-10-11T06:11:38.296+0000] {standard_task_runner.py:91} INFO - Job 963: Subtask load_ratings_to_gold
[2025-10-11T06:11:38.376+0000] {task_command.py:426} INFO - Running <TaskInstance: pipeline_flow.load_ratings_to_gold scheduled__1999-03-28T00:00:00+00:00 [running]> on host 321597a50810
[2025-10-11T06:11:38.532+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='oluwaseyiogundimu10@gmail.com' AIRFLOW_CTX_DAG_OWNER='seyi' AIRFLOW_CTX_DAG_ID='pipeline_flow' AIRFLOW_CTX_TASK_ID='load_ratings_to_gold' AIRFLOW_CTX_EXECUTION_DATE='1999-03-28T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__1999-03-28T00:00:00+00:00'
[2025-10-11T06:11:38.535+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-10-11T06:11:38.724+0000] {s3_client.py:30} INFO - âœ… Successfully initialized S3 client.
[2025-10-11T06:11:38.727+0000] {read_bucket.py:26} INFO - Reading ratings/1999-04.csv from movie-pipeline-silver bucket.
[2025-10-11T06:11:39.800+0000] {read_bucket.py:36} ERROR - Error reading ratings/1999-04.csv from movie-pipeline-silver: An error occurred (NoSuchKey) when calling the GetObject operation: The specified key does not exist.
[2025-10-11T06:11:39.803+0000] {load.py:113} WARNING - Partition ratings/1999-04.csv not found. Skipping.
[2025-10-11T06:11:39.804+0000] {load.py:120} WARNING - No ratings data found in selected partitions. Skipping load.
[2025-10-11T06:11:39.806+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-10-11T06:11:39.808+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-10-11T06:11:39.828+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=pipeline_flow, task_id=load_ratings_to_gold, run_id=scheduled__1999-03-28T00:00:00+00:00, execution_date=19990328T000000, start_date=20251011T061138, end_date=20251011T061139
[2025-10-11T06:11:40.118+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2025-10-11T06:11:40.157+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-10-11T06:11:40.161+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
